tests/debugger_tests/setup-debuggertests.cmd

```diff
@@ -51,7 +66,9 @@ REM ===
 REM === download debuggertests package
 REM ===
 REM =========================================================================================
-set DEBUGGERTESTS_URL=https://dotnetbuilddrops.blob.core.windows.net/debugger-container/Windows.DebuggerTests.zip
+REM TODO: change the archive name!!! (this is a test-only)
+REM TODO: name should be built from __OS and __Arch parameters.
```

Comment:
Just marking this todo with a comment for myself so I can confirm the final version resolved it